develop/how-to-contribute/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "How to Contribute to wxWidgets"
+---
+
+This short text tries to explain what should you do if you'd like to contribute to wxWidgets. wxWidgets is a real community project and its success depends heavily on your help and new developers are always warmly welcome!
+
+## Deciding How to Contribute
+
+This guide is addressed to code contributors, however this is not the only way to help wxWidgets and just because the other ones are not described in details here doesn't mean they are not, or less, important.  Contributions to documentation, including both fixing any problems in [the manual](https://docs.wxwidgets.org/stable/) included in wxWidgets itself and writing external tutorials/articles about wxWidgets, are always very much appreciated.  Testing wxWidgets, especially with lesser used compilers/architectures/operating systems is very important too. So is helping with answering the users questions on [wx-users mailing list](/support/mailing-lists/). And even [reporting bugs](/develop/how-to-report-issues/) is a welcome contribution too, although contributing the fixes for the bugs you discover is even better.
+
+However the most common kind of contribution remains help with improving wxWidgets itself, i.e. submitting code fixing bugs or implementing the new features for the library. If you would like to do this but are not sure what exactly would you like to do, please have a look at our [roadmap](/develop/roadmap/) listing the most important goals for the upcoming wxWidgets releases. And if this is not enough, there are many, many [open bugs](https://github.com/wxWidgets/wxWidgets/issues) fixes for which would be very welcome.
+
+Alternatively, you may need a feature for your program which isn't present in wxWidgets yet. In this case you're arguably the best person to implement it--you know exactly how the new feature should be used. And while you could implement support for this feature inside your own code, more often than not it is better to submit it to wxWidgets instead because this ensures that the feature will remain available in the future wxWidgets versions and you won't have to spend time upgrading your code for each new wxWidgets release. Of course, discussing the new code and submitting it for peer review results in better quality implementation.
```

Comment:
A space is used in other places around a dash (double hyphen)
```suggestion
Alternatively, you may need a feature for your program which isn't present in wxWidgets yet. In this case you're arguably the best person to implement it -- you know exactly how the new feature should be used. And while you could implement support for this feature inside your own code, more often than not it is better to submit it to wxWidgets instead because this ensures that the feature will remain available in the future wxWidgets versions and you won't have to spend time upgrading your code for each new wxWidgets release. Of course, discussing the new code and submitting it for peer review results in better quality implementation.
```